Get-Started.md

```diff
@@ -0,0 +1,111 @@
+# Tutorial
+- This tutorial will guide you from the very basics to real example of IPLD.
+- IPLD stands for the Inter Planetary Linked Data
+- When we get data from the internet, we know it comes from the right place, if the URL is https (some times not even! TODO: add symantec & fake certificates), but we don’t know if it was the data we intended to get!
+- The same problem is with links, we don’t know if a link is still linking to the right page
+- Finally, the general problem of the current web is that we trust the source to give us the right data, so other computers in the network cannot really help sharing the same data!
+- These 3 properties are secure linking, distributed naming and immutable content!
```

Comment:
These 3 properties are secure linking, distributed naming**,** and immutable content!